kite-data/kite-data-spark/pom.xml

```diff
@@ -318,5 +318,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
```

Comment:
We seem to have added this type of config to each child of kite-data except for flume and hbase. I know we don't use hbase, so if we don't use flume then that makes sense why those two were left out, but I thought we also don't use Spark. So why the change to this file?